enhancements/network/kubernetes-nmstate-operator-to-cvo.md

```diff
@@ -0,0 +1,195 @@
+---
+title: add-kubernetes-nmstate-operator-to-cvo
+authors:
+  - "@bcrochet"
+  - "@yboaron"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-03-16
+last-updated: 2021-03-16
+status: provisional
+see-also:
+  - "/enhancements/machine-config/mco-network-configuration.md"
+---
+
+# Add Kubernetes NMState Operator to CVO
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Add the Kubernetes NMState Operator to CVO.
+
+Currently, there are multiple methods of installing Kubernetes NMState into a cluster: 
+
+- CNV uses a [custom operator - CNAO](https://github.com/kubevirt/cluster-network-addons-operator#nmstate) that installs Kubernetes NMstate, among other network related items.
+The CNAO is deployed using [Hyperconverged Cluster Operator - HCO](https://github.com/kubevirt/hyperconverged-cluster-operator).
+To deploy Kubernetes NMstate it needs to deploy HCO --> CNAO --> Kubernetes NMstate
+- A [new operator](https://github.com/openshift/kubernetes-nmstate/blob/master/manifests/kubernetes-nmstate-operator.package.yaml) has been developed specifically to install Kubernetes NMstate using OLM.
+
+It is desirable for this to converge.
+
+Adding the Kubernetes NMstate as part of the CVO will allow for convenient and non-cumbersome use, which will also allow for uniformity between the various platforms (CNV, IPI-Baremetal).
+
+## Motivation
+
+### Goals
+
+* Provide a GA Kubernetes NMState Operator delivered in CVO for the use of OpenShift Virtualization at install time.
+* Provide a GA Kubernetes NMState Operator delivered in CVO for the use of OpenShift customers at install time.
```

Comment:
As mentioned above, I'd reword this to something like "Provide NMState APIs by default on $platforms"

We should also reference https://github.com/openshift/enhancements/pull/399 since IIUC the goal here is to provide the APIs enabled via that work by default, on some subset of platforms where they are frequently needed?